docs/development/design/3-DN-Retrieving_unique_graphs_from_Otel_data/json_data_converter_HOWTO.md

```diff
@@ -0,0 +1,330 @@
+# Configuring tel2puml/find_unique_graphs
+
+## Table of Contents
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li><a href="#1-introduction">1. Introduction</a></li>
+    <li><a href="#2-configuration-file-structure">2. Configuration File Structure</a></li>
+    <li><a href="#3-understanding-configuration-settings">3. Understanding Configuration Settings</a>
+      <ol>
+        <li><a href="#31-understanding-the-path-structure">3.1 Understanding the path structure</a></li>
+        <li><a href="#32-data_location">3.2 data_location</a></li>
+        <li><a href="#33-header">3.3 header</a></li>
+        <li><a href="#34-span_mapping">3.4 span_mapping</a></li>
+        <li><a href="#35-field_mapping">3.5 field_mapping</a></li>
+        <li><a href="#36-value_type-attribute">3.6 value_type attribute</a></li>
+      </ol>
+    </li>
+    <li><a href="#4-examples">4. Examples</a>
+      <ol>
+        <li><a href="#example-1-simple-span-attribute">Example 1: Simple span attribute</a></li>
+        <li><a href="#example-2-referencing-a-header-attribute">Example 2: Referencing a header attribute</a></li>
+        <li><a href="#example-3-complex-field-mapping">Example 3: Complex field mapping</a></li>
+      </ol>
+    </li>
+    <li><a href="#5-troubleshooting">5.0 Troubleshooting</a></li>
+  </ol>
+</details>
+
+
+
+## 1. Introduction
```

Comment:
Might be worth adding a bit about what the structure of the internal JSON schema is to provide context for the mapping